sematic/log_reader.py

```diff
@@ -276,7 +277,9 @@ def _get_latest_log_file(prefix, cursor_file) -> Optional[str]:
     # recall that for v1 logs, each log file contains ALL the logs from
     # the beginning of the run until the time that file was uploaded. So
     # the latest log file contains all the logs we have for the run.
-    log_files = S3Storage().get_child_paths(prefix)
+    storage = get_storage_plugins([LocalStorage])[0]
```

Comment:
The API for `get_storage_plugins` is:
```python
def get_storage_plugins(default: List[AbstractStorage]) -> List[AbstractStorage]:
```
`LocalStorage` is the default in case no plugin is configured, which is the case when users launch the app for the first time on their local machine `$ sematic start`.
When Sematic is deployed, the helm chart is configured to activate the `S3Storage` plugin, in which case `get_storage_plugins` will return `[S3Storage]`.